apps/src/acemode/mode-javascript_codeorg.js

```diff
@@ -0,0 +1,104 @@
+/* global ace */
```

Comment:
This diff looks much bigger than it is. I just added indentation where appropriate to make this "make sense"